### ISSUE TITLE
Code not running properly

### ISSUE THREAD

**Issue by GhostedPast** [formulahendry/vscode-code-runner#107]
Hi all,

I am currently having an issue when i run my python file by pressing F1 and then typing 'Run Code', i get this error:

Running the contributed command:'code-runner.run' failed.

If i go to run it again, it says "Code already running!"
and if i go to stop it running i get this error:

Running the contributed command:'code-runner.stop' failed.

I'm confused as to why this is happening at this point in time and any help would be greatly appreciated!

Cheers,
Ghost

**Comment by formulahendry** [comment]
What type of language are you running? Have you tired to run other files?

**Comment by GhostedPast** [comment]
I am running python, i have tried other files and other languages only to have the same outcome. Hope that helps

**Comment by formulahendry** [comment]
Could you reinstall the extension and try again?

**Comment by GhostedPast** [comment]
Yep, I've reinstalled it 3 or four times. Even reinstalled VSCode

**Comment by GhostedPast** [comment]
It's really bizarre because there are no script collisions or bugging that I can find.. it's like it doesn't recognize how to run it

**Comment by formulahendry** [comment]
Could you go to `Help` -> `Toggle Developer Tools`, and see if there is any error message in `Console`?

![image](https://cloud.githubusercontent.com/assets/1050213/25554347/a0af0a7e-2cfd-11e7-8f65-e53b6d7185c8.png)

**Comment by GhostedPast** [comment]
No errors... thats werid

**Comment by GhostedPast** [comment]
Oh now i see! I fixed it! It was clashing with a previous install that had not installed properly, it was running code based off the incomplete plugin. All i did was clear all of the plugin data, installed and not installed, reinstalled it and voila! All fixed.

I still find it strange that there was no console log of an error.

**Comment by formulahendry** [comment]
Closing this issue then.

**Comment by ilmoralito** [comment]
Now the problem described here occurred to me, it happens specifically with the php language, I have tried with javascript and it works correctly. I have reinstalled the plugin but the same behavior occurs, the version of code installed is 1.26.0. I enclose a screenshot of the log. Thanks for any suggestions.

![screenshot from 2018-08-14 12-31-00](https://user-images.githubusercontent.com/928920/44110924-4adce320-9fbe-11e8-9e9f-954daf34d9d3.png)
